ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -291,89 +310,136 @@ gcp_file_open(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
     return 0;
 }
 
+// POSIX remove, not supported for cloud objects.
 static int
 gcp_remove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, uint32_t flags)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(name);
-    WT_UNUSED(flags);
-
-    return 0;
+    std::cerr << "gcp_remove: object: " << name << ": file removal is not supported." << std::endl;
```

Comment:
This is potentially confusing as it indicates that the removal of the specific object is not supported. I believe you are trying to say basically the same thing as the comment above. I think that would be the appropriate message here.

Reply:
updated to not include object in the error message:
d11e042e49654d91f7ee8bd8450515ee77e1766e